src/dfmpreview/src/extension.ts

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import { workspace, window, ExtensionContext, commands, TextEditor, TextDocumentContentProvider, EventEmitter, Event, Uri, TextDocumentChangeEvent, ViewColumn,
+    TextEditorSelectionChangeEvent, TextDocument, Disposable } from "vscode";
+import * as fs from "fs";
+import * as path from "path";
+import * as child_process from 'child_process';
+
+
+let previewresult = "";
+let provider;
+let document_uri;
+let Is_end = true;
```

Comment:
One suggestion: follow typescript org's coding style
[https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)

---
In reply to: [73645944](https://github.com/dotnet/docfx/pull/558#discussion_r73645944) [](ancestors = 73645944)